thirdparty/cmdconfig/commands/cmdsource/cmdsource.go

```diff
@@ -29,6 +33,8 @@ func GetSourceRunner(name string) *SourceRunner {
 	}
 	c.Flags().StringVar(&r.FunctionConfig, "fn-config", "",
 		"path to function config file.")
+	c.Flags().BoolVar(&r.IncludeMetaResources,
+		"include-meta-resources", false, "include package meta resources in function input")
```

Comment:
what's function input? You either want to say:

1. Include in the _output_ of source?
2. Or if referring to the function that comes after the sink (there may not necessarily one one though): You mean `input items`: https://kpt.dev/book/02-concepts/03-functions

In general, need to more precision in describing entities.

Reply:
1) seems more appropriate here.

Reply:
@droot is right. Updated the description.